File: 2088-minimum-time-to-type-word-using-special-typewriter/2088-minimum-time-to-type-word-using-special-typewriter.java
class Solution {
    public int minTimeToType(String word) {
        char prev = 'a';
        int res = 0;
        for(char ch: word.toCharArray()){
            int diff = Math.abs(prev - ch);
            res += Math.min(diff, 26-diff);
            prev = ch;
        }
        res+= word.length();
        return res;
    }
}

File: 2088-minimum-time-to-type-word-using-special-typewriter/2088-minimum-time-to-type-word-using-special-typewriter.py
class Solution:
    def minTimeToType(self, word: str) -> int:
        res = len(word)
        curr = 'a'
        for ch in word:
            mn = min(26 - (abs(ord(ch) - ord(curr))), (abs(ord(ch) - ord(curr))))
            res+=mn
            curr = ch
        return res